### PR TITLE
Remove pedestal cleaning option in provenance tracking

### PR DESCRIPTION
**Pull request by Bultako**
action needed after #103

### CODE REVIEW

File: osa/provenance/utils.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def parse_variables(class_instance):
 
         class_instance.Analysisconfigfile_dl1 = os.path.realpath(configfile_dl1)
         class_instance.ObservationRun = class_instance.args[0].split(".")[0]
-        class_instance.PedestalCleaning = "True"
+        class_instance.PedestalCleaning = "False"
```
Comment (morcuended) [inline osa/provenance/utils.py:127]: Why False instead? Pedestal cleaning will be True by default

Comment (Bultako) [reply osa/provenance/utils.py:127]: ah sorry !
I though it was False by default.. then I see two options:
- completely remove PedestalCleaning from the provenance 
- close this PR.

Comment (morcuended) [reply osa/provenance/utils.py:127]: Is there a track of this config option in lstchain? If so I would say we could remove it from the osa provenance. In principle, lstosa will not set up this option from now on.

Comment (Bultako) [reply osa/provenance/utils.py:127]: actually there is no argument `pedestal-cleaning` aymore in `lstchain_dl1ab.py`
ok, I'm going to remove this from the provenance..

Comment (Bultako) [reply osa/provenance/utils.py]: done.

Comment (morcuended) [reply osa/provenance/utils.py]: There is no pedestal-cleaning argument anymore, but there is `--no-pedestal-cleaning` to disable it. I guess this info is kept somewhere in the metadata info of the files along with the config parameters used?

**Review by morcuended — APPROVED** [review]

File: osa/provenance/utils.py (hunk 1 above)
Comment (Bultako) [reply osa/provenance/utils.py]: It does not seem to be the case, all the content of the config file is kept, which means all the values for the thresholds :(

Anyhow, I think this is something that has to be done in `lstchain_dl1ab.py`, lstosa does not set up this option.

**Pull request merged**